housingbayarea/bloom
# refactor: public footer seeds (#4434)

This PR addresses https://github.com/bloom-housing/bloom/issues/4435


## Description

Pulls over the SiteFooter rewrite from core.

This one was lil weird because we have both Alameda and San Jose. The flow in Doorway was that for any customizations, we made them in CustomFooter file. Here, we have completely separate files for the footers, so I copied the CustomFooter component into those files and made the customizations there. With Alameda rolling into Doorway, this is temporary.

## How Can This Be Tested/Reviewed?

Check out the site footer and compare it to production. It should more closely match [dev](https://bloom.exygy.dev/) with [Alameda preview](https://deploy-preview-814--alameda-public-stage.netlify.app/) / [San Jose preview](https://deploy-preview-814--san-jose-public-stage.netlify.app/) content.

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
